docs/markdown/snippets/pkgconfig_break.md

```diff
@@ -0,0 +1,32 @@
+## Deprecation warning in pkg-config generator
+
+All libraries passed to the `libraries` keyword argument of the `generate()`
+method used to be associated with that generated pkg-config file. That means
+that any subsequent call to `generate()` where those libraries appear would add
+them into `Requires:` or `Requires.private:` field instead of `Libs:` or
```

Comment:
```suggestion
the filebase of the `generate()` that first contained them into `Requires:` or `Requires.private:` field instead of adding an `-l` to `Libs:` or
```